docs/can-canjs/can-api.md

```diff
@@ -117,6 +116,30 @@ class Owner extends ObservableObject {
 // -------------------------------
 // Define an observable Todo type:
 // -------------------------------
+class Todo extends ObservableObject {
+  static props = {
+    name: string,
+    complete: false
+  };
+
+    // `toggleComplete` is a method
+    toggleComplete() {
+        this.complete = !this.complete;
+    }
+}
+
+// Create a todo instance:
+const todo = new Todo({ name: "Learn Observables" });
+```
+@codepen
+
+<details>
+<summary>Define Observable and observable list types with [can-observable-object ObservableObject] and [can-observable-array ObservableArray]:</summary>
```

Comment:
Can you change this summary to say something like "Expand to see a larger example with numerous property behaviors...".